Vagrantfile

```diff
@@ -52,10 +52,13 @@ Vagrant.configure("2") do |config|
       fi
     SHELL
 
-    # Run playbook to setup /etc/hosts on target host
+    # Run vagrant specific playbook for preliminary setup
     host.vm.provision "ansible_local" do |ansible|
-      ansible.playbook = "configure_hosts.yml"
+      ansible.playbook = "configure_vagrant.yml"
       ansible.inventory_path = "environments/vm/inventory"
+      ansible.extra_vars = {
+        rhaptos_tunnel_ip: "10.0.10.1"
```

Comment:
Interesting. Why this address? (This is more curiosity from lack of knowledge than actual review)

Reply:
Good question -- so this relying on the private_network subnet defined (10.0.10/24) where the gateway IP of `.1` is (by convention) associated with the host's interface on the network.

Reply:
ah okay. so this is telling the deployment there is some other machine at 10.0.10.1 where it can download `dist.rhaptos.org` packages? And we use the VM host here because that's where we have the ability to tunnel through bastion2? I'm surprised that the `10.0.10.1` on the VM is the same as the VM host, on the other hand. For my sake, can you point out how the VMs and the host are connected via the private network?

Reply:
Sure -- so the private network breaks down as:

* 10.0.10.1 - Your host machine (the address of its interface on the private network)
* 10.0.10.2 - The `cnx-deploy` VM (the IP for the VM which executes the deployment via ansible)
* 10.0.10.10 - The `cnx-target` VM (the IP for the VM which runs the cnx stack)

Per your comment, the insertion of the entry in `/etc/hosts` in `cnx-target` will cause it to resolve `dist.rhaptos.org` to `10.0.10.1` (your host) where the tunnel runs and forwards the request.

LMK if that helps / answers your question.

Reply:
Thanks for the explanation!